ecms_base/features/custom/ecms_paragraphs/config/install/filter.format.embed.yml

```diff
@@ -10,7 +10,7 @@ filters:
   filter_html:
     id: filter_html
     provider: filter
-    status: true
+    status: false
```

Comment:
If the intention is to remove the HTML filter altogether, which I get, then the "allowed_html" attribute should be removed

Reply:
It is completely unchecked. This is just how the config gets output: 

![Screen Shot 2020-10-14 at 9 30 52 AM](https://user-images.githubusercontent.com/2349061/95995888-0add5b80-0e00-11eb-86eb-a183155bda1c.png)